fpdf/table.py

```diff
@@ -63,12 +65,15 @@ def __init__(
             line_height (number): optional. Defines how much vertical space a line of text will occupy
             markdown (bool): optional, default to False. Enable markdown interpretation of cells textual content
             text_align (str, fpdf.enums.Align): optional, default to JUSTIFY. Control text alignment inside cells.
+            v_align (str, fpdf.enums.AlignV): optional, default to CENTER. Control vertical alignment of cells content
             width (number): optional. Sets the table width
             wrapmode (fpdf.enums.WrapMode): "WORD" for word based line wrapping (default),
                 "CHAR" for character based line wrapping.
+            padding (number, tuple): optional. Sets the cell padding. Can be a single number or a sequence of numbers, default: half line height
         """
         self._fpdf = fpdf
         self._align = align
+        self._v_align = v_align
```

Comment:
```suggestion
        self._v_align = AlignV.coerce(v_align)
```
This will make sure we detect invalid values early, and also that `self._v_align` is always a `AlignV` enum (and not a string)